spec/round_spec.rb

```diff
@@ -0,0 +1,171 @@
+require './lib/card'
+require './lib/turns'
+require './lib/deck'
+require './lib/round'
+
+RSpec.describe Round do
+  it 'exists' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+
+    cards = [card_1, card_2, card_3]
+    deck = Deck.new(cards)
+    turns = Turns.new("Juneau", @card)
```

Comment:
Where does `@card` come from...? Remember, instance variables are based on the **scope** of a class or instance; another word for scope is "context". `@card` has no context in this file, because it belongs in the Deck class. So, instead of passing in `@card` in your tests, you should pass in `card_1` or one of the cards you made on lines 8-10.